CHANGELOG.md

```diff
@@ -38,7 +36,8 @@ All notable changes to this project will be documented in this file.
 -   #1690 : Add C support for list method `pop()`.
 -   #1877 : Add C and Fortran Support for set method `pop()`.
 -   #1917 : Add C and Fortran support for set method `add()`.
--   #1918 : Add C and Fortran support for set method `clear()`.
+-   #1918 : Add for set method `clear()`.
+-   #1918 : Add for set method `copy()`.
```

Comment:
I think this is missing `C and Fortran support`

Reply:
I didn't want to have 3 lines for every method that we add. This CHANGELOG is already going to be quite big so if we can limit it I think it will make it more readable.

I wanted to just put "added support" if support was added for all the languages that we support.

Maybe this is confusing though next to methods which already had Python support where the languages therefore need to be mentioned explicitly.

What do you think?

Reply:
I agree with you! I got your point when i was looking at the other PR for set union. I fancy `add support` better as well, but in this case it is also missing `support` , it's just `add for`

Reply:
Ah :sweat_smile: oops

Reply:
Fixed 47366167579c65883c3d87e9bf2648eb74da4bf7